sdk/common/src/test/java/io/opentelemetry/sdk/common/export/ConfigBuilderTest.java

```diff
@@ -59,6 +59,34 @@ void doubleProperty() {
     assertThat(doubleProperty).isEqualTo(5.6);
   }
 
+  @Test
+  void durationProperty() {
+    Long durationProperty =
+        ConfigBuilder.getDurationMillisProperty(
+            "duration", Collections.singletonMap("duration", "20"));
+    assertThat(durationProperty).isEqualTo(20L);
+    durationProperty =
+        ConfigBuilder.getDurationMillisProperty(
+            "duration", Collections.singletonMap("duration", "25ms"));
+    assertThat(durationProperty).isEqualTo(25L);
+    durationProperty =
+        ConfigBuilder.getDurationMillisProperty(
+            "duration", Collections.singletonMap("duration", "2s"));
+    assertThat(durationProperty).isEqualTo(2000L);
+    durationProperty =
+        ConfigBuilder.getDurationMillisProperty(
+            "duration", Collections.singletonMap("duration", "2m"));
+    assertThat(durationProperty).isEqualTo(120000L);
+    durationProperty =
+        ConfigBuilder.getDurationMillisProperty(
+            "duration", Collections.singletonMap("duration", "1h"));
+    assertThat(durationProperty).isEqualTo(3600000);
+    durationProperty =
+        ConfigBuilder.getDurationMillisProperty(
+            "duration", Collections.singletonMap("duration", "3d"));
+    assertThat(durationProperty).isEqualTo(259200000L); // 2592e8 = 3 * 24 * 3600 * 1000
```

Comment:
Instead of needing comments, we could use e.g., `TimeUnit.DAYS.toMillis` in these tests too for self-description.

Reply:
Done.